components/DownloadExcel.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div v-show="appStore.currentScenario.parameters" class="d-inline-block ms-auto">
```

Comment:
This is also still an issue - you can click the download button before there is any data to download.

Reply:
Sorry, yeah, this is actually an issue with v-show not working well with the tooltips I think (because of the teleport?). Replacing with v-if seems to work.